### PR TITLE
replace views with handles in shape, manifest and handle-connection.

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: runtime/manifest-parser.peg [hunk 1]
```diff
@@ -105,7 +105,7 @@ ManifestStorage
   {
     items = optional(items, extractIndented, []);
     return {
-      kind: 'view',
+      kind: 'store',
```
Comment (smalls) [inline runtime/manifest-parser.peg]: Store is the concrete storage system, handle is the pointer to that? Whereas before we called both a view?

Comment (mmandlis) [reply runtime/manifest-parser.peg]: Yes, that's correct.

The `kind` is internal between the manifest-parser and manifest, so no manifest file changes are needed, in manifest one would still use `view` (though i hope we'll eventually rename them too):
```
view MyThings of [Thing] in 'data/things.json'
```

Comment (smalls) [reply runtime/manifest-parser.peg]: Nice thanks! That makes sense. Maybe an issue for the manifest format work?

Comment (mmandlis) [reply runtime/manifest-parser.peg]: filed: https://github.com/PolymerLabs/arcs/issues/938

**Pull request merged**